scripts/fuel_infra_install.sh

```diff
@@ -12,5 +12,19 @@ salt '*' saltutil.sync_all
 
 sleep 5
 
+# Initialize the loopback device for LVM on all the Cinder volume nodes
+CINDER_LVM_FILE="/srv/cinder-volume"
+for CINDER_VOLUME_NODE in $(salt -C 'I@cinder:volume:enabled:True' --out txt test.true | sed 's/:.*$//'); do
+    if ! salt "$CINDER_VOLUME_NODE" service.status cinder-device-setup 2>/dev/null; then
```

Comment:
this should be handled by salt, can you add role device_setup in cinder service for this?